docs/UserGuide.md

```diff
@@ -133,69 +137,79 @@ Adds a person to the address book.
 **Output**<br>
 
 <img src="images/addCommand_output.png" width="800" style="border: 2px solid lightgrey; border-radius: 10px;">
+</box>
 
 [Return to Table of Contents](#medicontact)
-
 <br>
 <br>
 
-### Clearing all entries : `clear`
+### <i class="fa-solid fa-broom"></i> Clearing all entries : `clear`
 
-Clears all entries from the address book.
+Clears all patient entries from the address book.
+<box type="definition" icon=":fa-solid-spell-check:" light no-background>
 
 **Format**: `clear`
+</box>
 
-<box type="warning" seamless>
+<box type="warning" icon=":fa-solid-circle-info:" light no-background>
 
-**Warning**: After confirmation, any cleared data or contact cannot be retrieved. This action is IRREVERSIBLE.
+**Warning**: After confirmation, any cleared data or patient contact cannot be retrieved. This action is IRREVERSIBLE.
 
 </box>
 
 * Recommended practice: Always have a copy of current data. You may refer to [Exporting the data](#exporting-the-data). You can use [import feature](#importing-contacts-import) if you have lost your data after clearing it unintentionally.
-* You can directly delete the addressbook.json file which can be found in data folder or in the data directory if you want to recover the sample data.
+* You can directly delete the `addressbook.json` file which can be found in data folder or in the data directory if you want to recover the sample data.
+
+<box type="important" light no-background>
 
 **Confirmation Requirement:**
 
 * A confirmation window will appear before clearing all contacts in MediContact, 
 asking you to confirm if you are sure you want to proceed. 
-You must select “OK” to clear the address book. If you select “Cancel,” 
+You must select `OK` to clear the address book. If you select `Cancel`, 
 the action will be aborted, and the address book will remain unchanged.
-* Keyboard commands such as “Enter,” “Return,” and “Space” act as shortcuts to select the OK button, while “Esc” triggers the cancel button.
+* Keyboard commands such as `Enter`, `Return`, and `Space` act as shortcuts to select the OK button, while `Esc` triggers the cancel button.
 
 <img src="images/clearConfirmation.png" width="800">
+</box>
 
 [Return to Table of Contents](#medicontact)
-
 <br>
 <br>
 
 
-### Deleting a person : `delete`
+### <i class="fa-solid fa-user-slash"></i> Deleting a person : `delete`
 
-Deletes the specified person from the address book.
+Deletes the specified patient record from the address book.
+
+<box type="definition" icon=":fa-solid-spell-check:" light no-background>
 
 **Format**: `delete INDEX` / `delete NAME`
+</box>
 
-<box type="warning" seamless>
+<box type="warning" icon=":fa-solid-circle-info:" light no-background>
 
-**Warning**: After confirmation, any deleted data or contact cannot be retrieved. This action is IRREVERSIBLE.
+**Warning**: After confirmation, any deleted data or patient contact cannot be retrieved. This action is IRREVERSIBLE.
```

Comment:
I think patient's contact sounds better. But this is fine too if patient contact as a whole is treated as a thing

Reply:
I think patient contact as a whole is better cause I meant it as a whole list